tests/python/ConfigTest.py

```diff
@@ -842,12 +838,22 @@ def test_virtual_display(self):
                                              "sRGB")))
         self.assertEqual(1, len(cfg.getViews(OCIO.VIEW_SHARED, "sRGB")))
 
+        self.assertTrue(cfg.displayHasView("sRGB", "sview1"))
+        self.assertTrue(cfg.viewIsShared("sRGB", "sview1"))
```

Comment:
For all the functions that return booleans, please have a test where it returns true, another where it returns false, and a third where the arguments are bogus, e.g. a name that's either empty or not in the config.